src/components/App.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import Header from "./Header/Header";
+import Footer from "./Footer/Footer";
+import Body from "./Body/Body";
+import ErrorBoundary from "./ErrorBoundary";
+
+function App() {
+  return (
+    <div className="App">
+      <Header />
+      <ErrorBoundary>
+        <Body />
+      </ErrorBoundary>
```

Comment:
What happens if an error occurs in the Header or Footer section?
You need to wrap the App component in <ErrorBoundary> and you need to do this in the index.tsx file